feature1/one-more-file.txt

```diff
@@ -0,0 +1,3 @@
+This file was also added in the feature1 branch
```

Comment:
It's really good that you added this line of code

Reply:
Thanks for this comment, ayankelevich!

Reply:
No, I have to thank ayankelevich